app/_locales/es_419/messages.json

```diff
@@ -57,118 +72,175 @@
+  },
+  "affirmAgree": {
+    "message": "Acepto"
+  },
+  "aggregatorFeeCost": {
+    "message": "Cuota de red de agregador"
+  },
+  "alertDisableTooltip": {
+    "message": "Esto se puede modificar en \"Configuración > Alertas\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Explorando un sitio web con una cuenta no conectada seleccionada"
+  },
+  "alertSettingsUnconnectedAccountDescription": {
+    "message": "Esta alerta aparece en la ventana emergente cuando explora un sitio conectado de Web3, pero la cuenta seleccionada actualmente no está conectada."
+  },
+  "alerts": {
+    "message": "Alertas"
+  },
+  "alertsSettingsDescription": {
+    "message": "Habilitar o deshabilitar cada alerta"
+  },
+  "allowExternalExtensionTo": {
+    "message": "Permitir que esta extensión externa haga lo siguiente:"
+  },
+  "allowOriginSpendToken": {
+    "message": "¿Permitir que $1 gaste su $2?"
+  },
+  "allowThisSiteTo": {
+    "message": "Permitir que este sitio haga lo siguiente:"
+  },
+  "allowWithdrawAndSpend": {
+    "message": "Permitir que se retire $1 y gastar hasta el siguiente importe:"
   },
   "amount": {
-    "message": "Monto"
+    "message": "Importe"
+  },
+  "amountInEth": {
+    "message": "$1 ETH"
+  },
+  "amountWithColon": {
+    "message": "Importe:"
   },
   "appDescription": {
-    "message": "Una billetera de Ethereum en tu navegador",
-    "description": "The description of the application"
+    "message": "Una cartera de Ethereum en el explorador"
   },
   "appName": {
-    "message": "MetaMask",
-    "description": "The name of the application"
+    "message": "MetaMask"
+  },
+  "approvalAndAggregatorTxFeeCost": {
+    "message": "Cuota de red de agregador y aprobación"
+  },
+  "approvalTxGasCost": {
+    "message": "Costo de combustible por transacción de aprobación"
   },
   "approve": {
-    "message": "Aprobar"
+    "message": "Aprobar límite de gastos"
+  },
+  "approveSpendLimit": {
+    "message": "Aprobar límite de gastos de $1"
   },
   "approved": {
-    "message": "Aprobados"
+    "message": "Aprobado"
   },
   "asset": {
     "message": "Activo"
   },
+  "assets": {
+    "message": "Activos"
+  },
   "attemptToCancel": {
-    "message": "¿Intentas cancelar?"
+    "message": "¿Intentar cancelar?"
   },
   "attemptToCancelDescription": {
-    "message": "Este intento no garantiza que se cancele tu transacción original. Si el intento de cancelación es exitoso, se te cobrará la tasa de transacción que se muestra arriba."
+    "message": "Enviar este intento no garantiza que se cancelará la transacción original. Si el intento de cancelación se completa correctamente, se le cobrará la cuota de transacción anterior."
   },
   "attemptingConnect": {
-    "message": "Tratar de conectarse a blockchain."
+    "message": "Intentando una conexión a la cadena de bloques."
```

Comment:
Are we good with a literal translation of 'blockchain', or do we want it in English?

Reply:
my guess is "blockchain" is the word people will know